### PR TITLE
Add TSC members as maintainers

### PR DESCRIPTION
**Pull request by krisfreedain**
### Description
Add TSC members as MAINTAINERS - as discussed & approved in Oct 16 meeting

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dblock — CHANGES_REQUESTED** [review]
I've added the folks in CODEOWNERS as Maintainers. Once everyone accepted their pending invites we can merge this.

**Comment by dblock** [comment]
Screenshot 2024-10-27 at 10 59 50 AM

File: MAINTAINERS.md [hunk 1]
```diff
@@ -7,7 +7,21 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 
 ## Current Maintainers
 
-| Maintainer           | GitHub ID                                           | Affiliation |
-|----------------------|-----------------------------------------------------|-------------|
-| Kris Freedain        | [krisfreedain](https://github.com/krisfreedain)     | Amazon      |
-| Daniel Doubrovkine   | [dblock](https://github.com/dblock)                 | Amazon      |
+| Maintainer                | GitHub ID                                           | Affiliation |
+|---------------------------|-----------------------------------------------------|-------------|
+| Kris Freedain             | [krisfreedain](https://github.com/krisfreedain)     | Amazon      |
```
Comment (dblock) [inline MAINTAINERS.md:12]: @krisfreedain Sort this alphabetically?

**Review by reta — APPROVED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (krisfreedain) [reply MAINTAINERS.md]: done

**Review by dblock — APPROVED** [review]

**Pull request merged**